internal/git_fetch_test.go

```diff
@@ -0,0 +1,78 @@
+package internal
+
+import (
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/databricks/cli/internal/acc"
+	"github.com/databricks/cli/libs/dbr"
+	"github.com/databricks/cli/libs/git"
+	"github.com/databricks/cli/libs/vfs"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const examplesRepoUrl = "https://github.com/databricks/bundle-examples"
+const examplesRepoProvider = "gitHub"
+
+func assertGitInfo(t *testing.T, info git.GitRepositoryInfo, expectedRoot string) {
+	assert.Equal(t, "main", info.CurrentBranch)
+	assert.NotEmpty(t, info.LatestCommit)
+	assert.Equal(t, examplesRepoUrl, info.OriginURL)
+	assert.Equal(t, expectedRoot, info.WorktreeRoot.Native())
+}
+
+func TestFetchRepositoryInfoAPI(t *testing.T) {
```

Comment:
The current status is that integration tests need to be prefixed with `TestAcc` to run automatically.

Reply:
Good to know! updated. https://github.com/databricks/cli/pull/1945/commits/1df67d33de36b7c861d765d0c9840d621743892a